ckanext/ontario_theme/i18n/ckanext-ontario_theme.pot

```diff
@@ -996,13 +996,11 @@ msgstr ""
 
 #: ckanext/ontario_theme/templates/external/organization/snippets/helper.html:9
 #: ckanext/ontario_theme/templates/internal/organization/snippets/helper.html:9
-msgid "What are Organizations?"
+msgid "What are ministries?"
 msgstr ""
 
 #: ckanext/ontario_theme/templates/external/organization/snippets/helper.html:13
-msgid ""
-" <p> Find a list of Ontario Ministries here. Click on a ministry to see its "
-"datasets. </p> "
+msgid "Find a list of Ontario Ministries here. Click on a ministry to see its datasets."
 msgstr ""
```

Comment:
Please remove. This was part of the call out box on the left of the page that you have removed.